certified-connectors/CandidateZip/apiProperties.json

```diff
@@ -0,0 +1,28 @@
+{
+  "properties": {
+    "capabilities": [],
+    "connectionParameters": {
+      "token": {
+        "oAuthSettings": {
+          "clientId": "<client-id-please-replace>",
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://<replace with candidatezip authorization url>"
+            },            ,
```

Comment:
@CandidateZip Please remove the extra comma added after the value of authorizationUrl

Reply:
I submitted the changes.